_content/monographs/lamp-for-path-to-enlightenment_dipamkara-atisha.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Lamp for the Path to Enlightenment"
+authors:
+  - "Atīśa Dīpaṃkara Śrījñāna"
+editor: "Jamgön Kongtrul Lodrö Thaye"
+translator: "Khenpo David Karma Choephel"
+external_url: "https://dharmaebooks.org/the-lamp-for-the-path-to-enlightenment/"
+course: tibetan
+tags:
+  - path
+year: 2017
+month: nov
+page: 40
+publisher: "Dharma Ebooks"
```

Comment:
No publisher and move it to booklets I think.  Or maybe canon if this is considered canon by the Tibetans?